### ISSUE TITLE
strips all other meta data

### ISSUE THREAD

**Issue by fmp777** [dailymuse/s3-content-type-fixer#11]
I tested this on a few files and found that this script strips all other meta-data  on every object it updates. Probably not a good idea.

**Comment by ysimonson** [comment]
Yes, good point. Do you have a suggestion/PR in mind on how to preserve it?

**Comment by fmp777** [comment]
I'm not a python man, so I couldn't help you there.  Not sure if its the script causing the loss or a dependency.  I'd imagine reading meta data, preserving it, then passing it alone with new content-type would be what you want though?
